src/ui/page/home.html

```diff
@@ -52,7 +55,7 @@ <h2 class="heading-medium heading heading--red-top lpg-other-learning">
                 {#if plannedLearning.length}
                 <div class="resources lpg-other-courses">
                     {#each plannedLearning as course}
-                    <DisciteCourse course={course} modifier="home" /> {/each}
+                    <DisciteCourse course={course} modifier="home" today="{today}" /> {/each}
```

Comment:
See the other comment about `{today}`. Also with svelte v2 you can just do {today}, or today={today} (without the double `"`)